src/dataexplorer/dataexplorer-application/src/components/PatientInfo/DisplayPatientInfo.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { IPatient } from 'types/IPatient';
+import { Row, Col, Card, Table } from 'antd';
+import './PatientInfo.style.scss';
+
+const DisplayPatientInfo = (props: {
+    patient: IPatient;
+    handleClick: any;
+    dataSource: fhir.ResourceBase[];
+}) => {
+    const name =
+        props.patient.name[0].given[0] + ' ' + props.patient.name[0].family;
+    const columns = [
+        {
+            title: 'Skjemanavn',
+            dataIndex: 'schemaName',
+            key: 'schemaName',
+        },
+        {
+            title: 'Innsendt',
+            dataIndex: 'submitted',
+            key: 'submitted',
+        },
+        {
+            title: 'ID',
+            dataIndex: 'id',
+            key: 'id',
+        },
+    ];
+    console.log(props.patient);
+
+    return (
+        <>
+            <Row gutter={[1, 40]} justify="center">
+                <Col span={12}>
+                    <Card
+                        className="patient-card"
+                        type="inner"
+                        hoverable
+                        key={props.patient.id}
+                        title={name}
+                    >
+                        <div className="info-container">
+                            <div className="info-left">
+                                <h4>Pnr:</h4>
+                                <span>{props.patient.id}</span>
```

Comment:
Replace "Pnr" with "Personnummer? props.patient.id is not the same as personnummer. I would suggest passing this value from the parent component.